Services/DataBaseService/MigrationScripts/03.2020/02_ScriptsTableCreate.sql

```diff
@@ -0,0 +1,13 @@
+USE [TradingStation];
+
+IF OBJECT_ID('dbo.ExecutedScripts', 'U') IS NOT NULL
+    DROP TABLE [dbo].[ExecutedScripts]
+
+CREATE TABLE [dbo].[ExecutedScripts]
+    (
+        Id	    INT IDENTITY     NOT NULL,
+		FileName    NVARCHAR(100)     NOT NULL,
+        Code    NVARCHAR(MAX)    NOT NULL,
+        CONSTRAINT PKscripts
+            PRIMARY KEY CLUSTERED (Id),
+    );	
```

Comment:
May be it will be usefull if we can see when script was executed?

Reply:
Now we can: the column added, and execution time of each script is auto-added into it